resources/views/achievement/give.blade.php

```diff
@@ -0,0 +1,161 @@
+@extends('website.layouts.panel')
+
+@section('page-title')
+    Achievement Administration
+@endsection
+
+@section('panel-title')
+    Give an Achievement
+@endsection
+
+@section('panel-body')
+
+    <form method="post"
+          action="{{ route("achievement::give", ['id' => $achievement->id]) }}">
+
+        {!! csrf_field() !!}
+
+        <div class="form-group">
+            <label for="id">Member</label>
+            <div id="user-select">
+                <div class="col-sm-10">
+                    <input type="text" class="form-control" id="member-name" placeholder="John Doe" required>
+                    <input type="hidden" id="member-id" name="user_id" required>
+                </div>
+                <div class="col-sm-2">
+                    <input type="button" class="form-control btn btn-success" id="member-clear" value="Clear">
+                </div>
+            </div>
+        </div>
+
+        <hr>
+
+        <strong>Achievement to give:</strong>
+        <li class="list-group-item achievement {{ $achievement->tier }}">
+
+            <div class="achievement-icon">
+                @if($achievement->image)
+                    <img src="{!! $achievement->image->generateImagePath(700,null) !!}" alt="">
+                @else
+                    No icon available
+                @endif
+            </div>
+            <div>
+                <strong>{{ $achievement->name }}</strong>
+                <p>{{ $achievement->desc }}</p>
+            </div>
+
+        </li>
+
+        @endsection
+
+        @section('panel-footer')
+
+            <button type="submit" class="btn btn-success pull-right" style="margin-left: 15px;">Give</button>
+
+            <a href="{{ route("achievement::list") }}" class="btn btn-default pull-right">Cancel</a>
+
+    </form>
+
+@endsection
+
+@section('stylesheet')
```

Comment:
Deze CSS moet in een LESS file, zie de pinned message van Bouke in de commissieslack.
 Add a line note